ci/code_checks.sh

```diff
@@ -109,9 +109,12 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     # Imports - Check formatting using isort see setup.cfg for settings
     MSG='Check import format using isort ' ; echo $MSG
-    isort --recursive --check-only pandas asv_bench
+    if [[ "$AZURE" == "true" ]]; then
+        isort --recursive --check-only pandas asv_bench | awk -F ':' '{print $0"\n" "##vso[task.logissue type=error;sourcepath=%(path)s]%(text)s"}'
+    else
+        isort --recursive --check-only pandas asv_bench | awk -F ':' '{print}'
+    fi
     RET=$(($RET + $?)) ; echo $MSG "DONE"
-
```

Comment:
in general try to avoid unrelated changes, even if they are small like this.

Reply:
Undid the change!